### PR TITLE
Applies a workaround to fix package-lock.json generation

### PR DESCRIPTION
**Pull request by josemigallas**
**What this PR does / why we need it**:

Due to a well-known [error in the npm registry](https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/16), `package-lock.json` is constantly being updated, changing tarball urls from https to http.

This workaround simply replaces all http back to https after lock file is generated. The reason why using `replace` is environment compatibility and ease of use.

Although hacky, this has been pointed out as a secure workaround by npm [(source)](https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/3).

**Verification**

Run `npm i`, there should be no changes.

### CODE REVIEW

**Comment by josemigallas** [comment]
Looks like it's a well-known issue with the registry -> https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/16

TLDR: safe workaround is manually swaping `http` -> `https`.

**Comment by hallelujah** [comment]
@josemigallas So just rebasing because it conflicts. About the image build, well we have docker-build job so it is fine
